.github/workflows/unit-tests.yml

```
@@ -0,0 +1,34 @@
+name: tests
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: |
+          sudo python3 -m pip install --upgrade pip
+          pip install -U setuptools
+          sudo pip3 install nbconvert pelican pytest-tornasync
```

```suggestion
          python3 -m pip install --upgrade pip
          pip3 install -U setuptools
          pip3 install nbconvert pelican pytest-tornasync
```

With setup-python providing the python sudo shouldn't be required anymore